src/rust/node-identifier/src/main.rs

```diff
@@ -0,0 +1,185 @@
+use futures::StreamExt;
+use grapl_config::env_helpers::FromEnv;
+use kafka::{
+    StreamProcessor,
+    StreamProcessorError,
+};
+use opentelemetry::{
+    global,
+    sdk::propagation::TraceContextPropagator,
+};
+use rusoto_dynamodb::DynamoDbClient;
+use rust_proto_new::graplinc::grapl::{
+    api::graph::v1beta1::{
+        GraphDescription,
+        IdentifiedGraph,
+    },
+    pipeline::{
+        v1beta1::Metadata,
+        v1beta2::Envelope,
+    },
+};
+use tracing::instrument::WithSubscriber;
+use tracing_subscriber::{
+    prelude::*,
+    EnvFilter,
+};
+
+mod dynamic_sessiondb;
+mod error;
+mod node_identifier;
+mod sessiondb;
+mod sessions;
+
+use crate::{
+    dynamic_sessiondb::NodeDescriptionIdentifier,
+    error::NodeIdentifierError,
+    node_identifier::NodeIdentifier,
+    sessiondb::SessionDb,
+};
+
+#[tokio::main]
+async fn main() -> Result<(), NodeIdentifierError> {
+    let (non_blocking, _guard) = tracing_appender::non_blocking(std::io::stdout());
```

Comment:
I am a bit surprised why we're not using init_grapl_env!

Reply:
I've been trying to eliminate that dependency as we go along because it hauls in a ton of sqs baggage. Unfortunately, it seemed like overkill here as setting up DynamoDB without it seemed like more trouble than it was worth. Eventually when we move to Scylla we won't (I dearly hope) any longer have a DynamoDB dependency, though.